blazor/maps/accessibility.md

```diff
@@ -39,7 +38,7 @@ The accessibility compliance for the Maps component is outlined below.
 
 ## WAI-ARIA attributes
 
-To meet accessibility standards, the Maps component follows to the [WAI-ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/alert/) patterns. In the Maps component, the following ARIA attributes are used:
+To meet accessibility standards, the Blazor Maps component follows to the [WAI-ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/alert/) patterns. In the Maps component, the following ARIA attributes are used:
```

Comment:
"WAI-ARIA" in second line is missing. The link or code style not provided.

Reply:
Hi @sabari-senthamaraikannan ,

I modified the reference link.

Regards,
@Backiaraj